source/core.py

```diff
@@ -207,6 +207,23 @@ def _setInitialFocus():
 	except:
 		log.exception("Error retrieving initial focus")
 
+
+def getWxLang(lang):

```

Comment:
Please add typing info here, you might want to double check the types, but I think it should be like this:

``` python
from typing import Optional
from wx import LanguageInfo
...
def getWxLang(lang: str) -> Optional[LanguageInfo]:

```

Reply:
yep that looks right, I'm guess the practice is to add these as function headers are created/updated?

Reply:
Yes please if it is possible. Sometimes the types are very dynamic and it can be quite a pain. Those situations are probably the most valuable to be able to narrow it down, but also the most time consuming. You can always do something like:
``` python
Union[ SomeConcreteType, SomeOtherType, Any ] # Any: Not all types are known yet, please update.
```